nle/env/base.py

```diff
@@ -521,39 +568,8 @@ def render(self, mode="human"):
         """
         chars_index = self._observation_keys.index("chars")
         if mode == "human":
-            message_index = self._observation_keys.index("message")
-            message = bytes(self.last_observation[message_index])
-            print(message[: message.index(b"\0")])
-            try:
-                inv_strs_index = self._observation_keys.index("inv_strs")
-                inv_letters_index = self._observation_keys.index("inv_letters")
-
-                inv_strs = self.last_observation[inv_strs_index]
-                inv_letters = self.last_observation[inv_letters_index]
-                for letter, line in zip(inv_letters, inv_strs):
-                    if np.all(line == 0):
-                        break
-                    print(
-                        letter.tobytes().decode("utf-8"), line.tobytes().decode("utf-8")
-                    )
-            except ValueError:  # inv_strs/letters not used.
-                pass
-            colors_index = self._observation_keys.index("colors")
-            chars = self.last_observation[chars_index]
-            colors = self.last_observation[colors_index]
-            rows, cols = chars.shape
-            nh_HE = "\033[0m"
-            BRIGHT = 8
-            for r in range(rows):
-                for c in range(cols):
-                    # cf. termcap.c.
-                    start_color = "\033[%d" % bool(colors[r][c] & BRIGHT)
-                    start_color += ";3%d" % (colors[r][c] & ~BRIGHT)
-                    start_color += "m"
-
-                    print(start_color + chr(chars[r][c]), end=nh_HE)
-                print("")
-            return
+            obs = self.last_observation
+            return self.tty_render(obs, print_guides=False)
```

Comment:
can I suggest either 
```suggestion
            self.tty_render(obs, print_guides=False)
            return 
```

or 

```suggestion
            image = self.tty_render(obs, print_guides=False)
            print(image)
            return 
```

The current API suggests render doesnt return anything, and so by writing it as either of these examples, we can clearly see that the printing must happen in the `tty_render` function, and not elsewhere in the code (ie perhaps with the returned value). 

in fact I would be tempted to go for the second example, remove the print function from the tty_render method, and rename it as "tty_to_string" or "get_tty_string". That way it can be programatically reused, possibly for logging by downstream users.